docs-chef-io/content/server/upgrade_server_ha_v2.md

```diff
@@ -163,13 +163,160 @@ Upgrading from Chef Backend 1.x to Chef Backend 2.x requires full cluster downti
 
 ## Chef Backend 2.x to 3.x Upgrade
 
-| Running Version | Upgrade To Version | Upgrade Type |
-|---------|---------|------|
-| 2.x | 2.x | Rolling |
-| 2.1 | 3.0 | Rolling |
-| 2.0 | 3.0 | Downtime |
+### Planning
+
+The versions of Chef Backend, Elasticsearch, and Chef Infra Server running your system determines the type of upgrade, index treatment, and steps for your upgrade.
+
+#### Upgrade Type
+
+Systems with Elasticsearch 5.4.1 or lower require downtime. Systems with Elasticsearch 5.6.16 or higher use a rolling upgrade.
+
+| Running Backend Version | Upgrade To Version | Elasticsearch Version | Upgrade Type |
```

Comment:
This is not the elasticsic version, it is the version of elasticsearch that created the indices, those 2 can be different if the customers have done a rolling upgrade previously without updating the indices.